app/models/search-result.ts

```diff
@@ -73,11 +73,15 @@ export default class SearchResultModel extends Model {
         return this.resourceMetadata['@id'];
     }
 
+    // returns list of affilated institutions for users
     // returns list of contributors for osf objects
     // returns list of affiliated institutions for osf users
     get affiliatedEntities() {
         if (this.resourceType === 'user') {
-            // return something
+            if (this.resourceMetadata.affiliation) {
+                return this.resourceMetadata.affiliation.map((item: any) =>
+                    ({ name: item.name[0]['@value'], absoluteUrl: item['@id'] }));
```

Comment:
this assumes each institution has exactly one one `name`... with how the osf backend gathers metadata at the moment, that's reasonably safe, but i'd like to nudge toward allowing most every metadata property to have multiple values which can all be seen by the user

(seeing the single-value assumption (`[0]`) often in the search frontend -- seems fine to leave this as-is for consistency and maybe talk about shifting to dynamic multi-value properties if/when we want searchable cedar)

Reply:
Yea, this was one that I wasn't too sure how to best handle while implementing this earlier in the search project. At the moment, there seems to typically be just one `name`, but would like to be able to show all of them, without it being overwhelming